### DISCUSSION TITLE
Repo for keymaster-public to deploy hubs in Canada

### DISCUSSION THREAD

**Discussion by agriggs2** [mozilla/hubs#5650] [Hubs Cloud Questions] [answered]
I have a client that needs to keep their data in Canada, I have been able to deploy nearspark, photomnemonic, speelycaptor in ca-central-1 but when I checked the keymaster-public application on us-east its pointing to a "dead" repo (its 404'ing for me) https://github.com/mozillareality/keymaster. 

Is this code available somewhere for me to deploy it onto ca-central-1?

Thanks

**Comment by pattersonbl2** [comment]
Sorry, The source code for the key master is in a private repo due to the nature of its function.  We currently don't support this type of deployment situation.  We hope to in the future open more sections of the code to the public for custom deployments.

**Comment by gfodor** [comment, marked as answer]
Sorry to chime in here (ex-moz, wrote keymaster) - but to be clear that
lambda at the time I authored it is not relying on security through
obscurity and wasn’t started in a private repo for security reasons. I
forget if the code is minified or not, but the lambda is obviously public
so you should be able to see the code via the AWS APIs. Not sure if
anything has changed but wanted to clarify.

**Reply by pattersonbl2** [reply]
A better answer than what I could research.  I will mark this for future reference. Thank you, Greg !!

**Comment by gfodor** [comment]
Sure thing!
